backend/app/api.py

```
@@ -2304,30 +2304,53 @@ def get_item_groups(current_user, game_name, item_name):
-        # Query all group members for this game to filter in memory safely
+        checksum = request.args.get('checksum') or request.args.get('datapackage_checksum')
+        if not checksum:
+            room_db_id = request.args.get('room_db_id')
+            if room_db_id:
+                try:
+                    room = session.query(TrackedRoom.game_checksums_json).filter_by(id=int(room_db_id)).first()
+                    if room and room[0]:
+                        checksums = json.loads(room[0])
+                        checksum = checksums.get(game_name)
+                except Exception:
+                    pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)